Offboarding/delete_accounts.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import json
+import pandas as pd
+import numpy as np
+import sys
+import logging
+from pathlib import Path
+logging.basicConfig(filename='logging_for_deleting_accounts.log',level=logging.INFO)
+
+"""
+1) This python script is used to delete accounts based on the list of accounts provided in a csv file named 'accounts_list'.  Please see example datafile for formatting.
+2) For the data file, please define 2 columns:
+    Column 1: the API URI for your region in CloudCheckr
+        Valid regions:
+            https://api.cloudcheckr.com
+            https://eu.cloudcheckr.com
+            https://au.cloudcheckr.com
+            https://gov.cloudcheckr.com
+    Column 2: the corresponding account name of the account you wish to be deleted (as it appears in the CloudCheckr landing page)
+    Note:  The first row is for column names
+3) To call this script, use the following: python3 delete_accounts.py <admin level access key>
+4) Please ensure that this file is in the same folder/directory as the data file.
+"""
+
+def log_information(log_string):
+    logging.info(log_string)
+
+
+def write_users_created(users_output, users_recorder):
+    np.savetxt(users_output, users_recorder, fmt="%s", delimiter=",", newline="\n")
+
+
+def get_accounts_input(file_name):
+
+    accounts_list = pd.read_csv(file_name)
+
+    return accounts_list
+
+def delete_account(env, admin_api_key, account_name):
+
+    api_url = env + "/api/account.json/delete_account"
+
+    delete_user_info = json.dumps({"account_name": account_name})
+
+    r7 = requests.post(api_url, headers = {"Content-Type": "application/json", "access_key": admin_api_key}, data = delete_user_info)
```

Comment:
Would be good to wrap this in a try/catch block. If one API call fails, it should log the error and continue on.

Reply:
Success will log 200 + the endpoint and request parameters to identify which data row corresponds with the request.  

Failure will log the endpoint + status code, the request parameters, and hopefully return an ident message for support to debug.